apps/api/src/tests/db/index.ts

```diff
@@ -18,29 +18,35 @@
  */
 
 import assert from 'node:assert';
-import { after, describe, it } from 'node:test';
+import { after, before, describe, it } from 'node:test';
 
+import { PostgreSqlContainer, StartedPostgreSqlContainer } from '@testcontainers/postgresql';
 import { eq } from 'drizzle-orm';
-import { drizzle } from 'drizzle-orm/node-postgres';
 
+import { initMigration, startDb, type PostgresDb } from '../../db/index.ts';
 import { actions } from '../../db/schemas/actions.ts';
 import { agreements } from '../../db/schemas/agreements.ts';
 import { applicationContents } from '../../db/schemas/applicationContents.ts';
-import { applications } from '../../db/schemas/applications.ts';
 import { collaborators } from '../../db/schemas/collaborators.ts';
 import { files } from '../../db/schemas/files.ts';
 import { revisionRequests } from '../../db/schemas/revisionRequests.ts';
 
-import { connectionString } from '../../../drizzle.config.ts';
-
-type PostgresDb = ReturnType<typeof drizzle>;
-
 describe('Postgres Database', () => {
 	let db: PostgresDb;
+	let container: StartedPostgreSqlContainer;
+
+	const user_id = 'testUser@oicr.on.ca';
+
+	before(async () => {
+		container = await new PostgreSqlContainer().start();
+		const connectionString = container.getConnectionUri();
+		db = startDb(connectionString);
+
+		await initMigration(db);
+	});
 
```

Comment:
These are tests from the previous PR which do not have a Service yet (test plain Drizzle actions)